src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx

```diff
@@ -75,43 +60,43 @@ keras_model.compile(optimizer="rmsprop", loss="mse", metrics=["accuracy"])
 mlflow.keras.autolog()
 results = keras_model.fit(
     x_train, y_train, epochs=20, batch_size=128, validation_data=(x_val, y_val))
-# ...`,
-    },
-    {
-      title: 'Integrate KerasTuner',
-      docsLink: DOCUMENTATIONS.INTEGRATIONS.KERAS_TUNER,
-      code: `from aim.keras_tuner import AimCallback
-
-# ...
-tuner.search(
-    train_ds,
-    validation_data=test_ds,
-    callbacks=[AimCallback(tuner=tuner, repo='.', experiment='keras_tuner_test')],
-)
 # ...`,
     },
     {
       title: 'Integrate XGBoost',
-      docsLink: DOCUMENTATIONS.INTEGRATIONS.XGBOOST,
-      code: `from aim.xgboost import AimCallback
+      code: `from sklearn import datasets
+from sklearn.model_selection import train_test_split
+import mlflow.xgboost
+import xgboost as xgb
 
-# ...
-aim_callback = AimCallback(repo='/path/to/logs/dir', experiment='experiment_name')
-bst = xgb.train(param, xg_train, num_round, watchlist, callbacks=[aim_callback])
-# ...`,
-    },
-    {
-      title: 'Integrate CatBoost',
-      docsLink: DOCUMENTATIONS.INTEGRATIONS.CATBOOST,
-      code: `from aim.catboost import AimLogger
+# Set FastTrackML tracking server
+mlflow.set_tracking_uri("${fasttrack_server}")
 
 # ...
-model.fit(train_data, train_labels, log_cout=AimLogger(loss_function='Logloss'), logging_level="Info")
+# prepare train and test data
+iris = datasets.load_iris()
+X = iris.data
+y = iris.target
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
+
+# enable auto logging
+mlflow.xgboost.autolog()
+
+dtrain = xgb.DMatrix(X_train, label=y_train)
+dtest = xgb.DMatrix(X_test, label=y_test)
+
+with mlflow.start_run():
+    # train model
+    params = {
+        "objective": "multi:softprob",
+        "num_class": 3,
+        # ...
+    }
+    model = xgb.train(params, dtrain, evals=[(dtrain, "train")])
```

Comment:
Please make this sample similar to the other ones by focusing only on `import`, `set_tracking_uri`, `autolog`, `start_run`, and `train`. Most other stuff should be replaced by ellipses. Users interested in XGBoost integration already know how to use XGBoost — they wanna know how to log their runs to FML.

Reply:
done